client/cli/go/cmds/volume.go

```diff
@@ -142,7 +142,7 @@ var volumeCreateCommand = &cobra.Command{
 		if kubePv && kubePvEndpoint == "" {
 			fmt.Fprintf(stderr, "--persistent-volume-endpoint must be provided "+
 				"when using --persistent-volume\n")
-			return fmt.Errorf("Missing endpoing")
+			return fmt.Errorf("Missing endpoint")
```

Comment:
This change is also in #373 , you probably want to cancel that one or remove this one.